samples/bluetooth/direct_test_mode/sysbuild.cmake

```diff
@@ -20,13 +20,6 @@ if(SB_CONFIG_SUPPORT_APPCORE AND DEFINED SB_CONFIG_APPCORE_IMAGE_NAME)
   set_property(GLOBAL APPEND PROPERTY PM_CPUAPP_IMAGES "${SB_CONFIG_APPCORE_IMAGE_NAME}")
   set_property(GLOBAL PROPERTY DOMAIN_APP_CPUAPP "${SB_CONFIG_APPCORE_IMAGE_NAME}")
   set(CPUAPP_PM_DOMAIN_DYNAMIC_PARTITION ${SB_CONFIG_APPCORE_IMAGE_NAME} CACHE INTERNAL "")
-
-  string(REPLACE " " ";" shield_list "${SHIELD}")
```

Comment:
This should stay, but be changed to add the overlay for example for remote_hci with FEM. This will break.